drivers/vulkan/vulkan_context.cpp

```diff
@@ -234,6 +234,43 @@ Error VulkanContext::_create_validation_layers() {
 	return OK;
 }
 
+typedef VkResult (*_vkEnumerateInstanceVersion)(uint32_t *);
+
+void VulkanContext::_obtain_vulkan_version() {
+	// init to Vulkan 1.0
+	vulkan_major = 1;
+	vulkan_minor = 0;
```

Comment:
We already init those to `1` and `0` in the header, so maybe it shouldn't be done here too (so we don't have two places to update or miss one of them if we decide to change init). You can move the comment in the header to make it clear that it's `1` and `0` on purpose.